.github/workflows/Coverage.yml

```diff
@@ -0,0 +1,41 @@
+name: Jest Coverage
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - name: Install dependencies
+        run: |
+          cd app
+          npm install
+
+      - name: Run Jest Tests and Generate Coverage Badge
```

Comment:
We can rename this to Generate Jest coverage badge

Reply:
done